performance/run.js

```diff
@@ -0,0 +1,12 @@
+const { execSync } = require('child_process');
+
+if(process.platform === "darwin") {
+    execSync('docker run --rm -v "$(pwd)":/sitespeed.io sitespeedio/sitespeed.io:12.2.0 --config=config/config-chrome.json http://host.docker.internal:8080/performance/performance.html', {stdio: 'inherit', cwd: __dirname});
```

Comment:
Misschien de versie van sitespeed.io ook configureerbaar maken. Nu zitten we op 12.2.0, maar dat blijkt maar redelijk diep in de code. Misschien kan dit best meegegeven worden via het NPM script.

Reply:
Lijkt me een nice to have.